HospitalToday/UI/MenuHandler.cs

```diff
@@ -0,0 +1,555 @@
+            var answer = Console.ReadLine();
+            int docId;
+            var result = int.TryParse(answer, out docId);
+            if (!result)
+            {
+                Console.WriteLine("Incorrect input!");
+            }
+            Person doctor = doctors.GetItemById(docId);
+
+            answer = Console.ReadLine();
+            int patId;
+            result = int.TryParse(answer, out patId);
+            if (!result)
+            {
+                Console.WriteLine("Incorrect input!");
+            }
+            Person patient = patients.GetItemById(patId);
+
+            var reportMedicines = GetListMedicines();
+
+            doctors.CreateReport(doctor, patient, reportMedicines, DateTime.Now);
+
+        }
+
+        private List<Medicine> GetListMedicines()
+        {
+            List<Medicine> meds = new List<Medicine>();
+            while (true)
+            {
+                Console.WriteLine("Medicine id:");
+                var answer = Console.ReadLine();
+                int id;
+                var result = int.TryParse(answer, out id);
+                if (!result)
+                {
+                    Console.WriteLine("Incorrect input!");
+                    break;
+                }
+
+                var medicine = medicines.GetItemById(id);
+                if (medicine != null)
+                {
+                    meds.Add(medicine);
+                }
+
+                Console.WriteLine("Add one more Item? - press 0");
+                answer = Console.ReadLine();
+                if(answer != "0")
+                {
+                    break;
+                }
+            }
+            return meds;
+        }
+
+        private void RemoveReport()
+        {
+            Console.Clear();
+            Console.WriteLine();
+            Console.WriteLine("Id:");
+            int id;
+            var answer = Console.ReadLine();
+            var result = int.TryParse(answer, out id);
+            if (result)
+            {
+                var report = reports.GetItemById(id);
+                if (report != null)
+                {
+                    reports.Remove(report);
+                }
+            }
+        }
+
+        private void GetReportById()
+        {
+            Console.Clear();
+            Console.WriteLine("Id:");
+            int id;
+            var answer = Console.ReadLine();
+            var result = int.TryParse(answer, out id);
+            if (result)
+            {
+                var report = reports.GetItemById(id);
+                if (report != null)
+                {
+                    var doctor = doctors.GetItemById(report.DoctorId);
+                    var patient = patients.GetItemById(report.PatientId);
+                    if (doctor == null || patient == null)
+                    {
+                        Console.WriteLine($"{report.Id} {report.Date}");
+                        Console.WriteLine($"{doctor.FirstName} {doctor.LastName}");
```

Comment:
Let's create some extension method for doctor and patient to display this data as a single string

Reply:
Extension methods were added.